Sources/StructuredQueriesCore/Operators.swift

```diff
@@ -54,6 +54,11 @@ extension QueryExpression where QueryValue: QueryBindable {
     BinaryOperator(lhs: self, operator: "=", rhs: other)
   }
 
+  public func eq(_ other: QueryValue) -> some QueryExpression<Bool>
+  where QueryValue: FixedWidthInteger {
+    BinaryOperator(lhs: self, operator: "=", rhs: other)
+  }
+
```

Comment:
I think this snuck in. Wanna move to another PR and include the other operators for consideration?

Reply:
yeah thought I reverted that. done...